source/FreeRTOS_TCP_Transmission_IPV6.c

```diff
@@ -89,7 +89,7 @@
                                   uint32_t ulLen,
                                   BaseType_t xReleaseAfterSend )
     {
-        TCPPacket_t * pxTCPPacket = NULL;
+        TCPPacket_IPv6_t * pxTCPPacket = NULL;
```

Comment:
Can we retain the older structure "TCPPacket_t" as this is going to break the unit tests.